src/Common/FitOnFhir.Common/Services/TokenValidationService.cs

```diff
@@ -64,16 +64,13 @@ public async Task<bool> ValidateToken(HttpRequest request, CancellationToken can
                 return false;
             }
 
-            // Read the token into the handler.
+            // Read the token
             _jwtSecurityTokenHandlerProvider.SetMapInboundClaims(false);
-            JwtSecurityToken jwtSecurityToken;
-            try
-            {
-                jwtSecurityToken = _jwtSecurityTokenHandlerProvider.ReadJwtToken(token);
-            }
-            catch (Exception e)
+            var jwtSecurityToken = _jwtSecurityTokenHandlerProvider.ReadJwtToken(token);
+
+            if (jwtSecurityToken == default)
             {
-                _logger.LogError(e, "The request JWT is malformed.");
+                _logger.LogError("The JWT token is empty.");
```

Comment:
nit: I don't think you need to log this the `_jwtSecurityTokenHandlerProvider` already logs the error.